docs/admin_guide/french_specifities/assets/dcat_data_gouv_fr_output.patch

```diff
@@ -0,0 +1,307 @@
++      <!-- geOrchestra change: use a 'nice' HTML view for the md description url-->
++      <rdf:Description rdf:about="{$url}/srv/fre/catalog.search#/metadata/{$uuid}">
+         <dct:format>
+           <dct:IMT>
+             <rdf:value>text/html</rdf:value>
+@@ -181,17 +183,25 @@
+           <dcat:size></dcat:size>
+         -->
+ 
+-          <xsl:if test="(gmd:protocol/gmx:Anchor/@xlink:href)[1]!=''">
++        <xsl:if test="(gmd:protocol/gmx:Anchor/@xlink:href)[1]!=''">
+           <dcat:mediaType>
+             <xsl:attribute name="rdf:resource" select="(gmd:protocol/gmx:Anchor/@xlink:href)[1]"/>
+             <xsl:value-of select="(gmd:protocol/gmx:Anchor)[1]"/>
+           </dcat:mediaType>
+-          </xsl:if>
+-          <xsl:if test="(gmd:protocol/gco:CharacterString)[1]!=''">
++        </xsl:if>
++
++        <!-- geOrchestra addition: put the description of the attached file -> fills the 'Description des données' field in uData -->
++        <xsl:if test="(gmd:description/gco:CharacterString)[1]!=''">
++          <dct:description>
++            <xsl:value-of select="(gmd:description/gco:CharacterString)[1]"/>
++          </dct:description>
++        </xsl:if>
++
++        <xsl:if test="(gmd:protocol/gco:CharacterString)[1]!=''">
+           <dct:format>
+             <xsl:value-of select="(gmd:protocol/gco:CharacterString)[1]"/>
+           </dct:format>
+-          </xsl:if>
++        </xsl:if>
+ 
+       </dcat:Distribution>
+     </xsl:for-each-group>
+@@ -273,7 +283,11 @@
+   -->
+   <xsl:template match="gmd:MD_DataIdentification|*[contains(@gco:isoType, 'MD_DataIdentification')]"
+                 mode="to-dcat">
+-    <dcat:Dataset rdf:about="{$resourcePrefix}/datasets/{iso19139:getResourceCode(../../.)}">
++
++    <!-- SIB addon-->
++    <!-- <dcat:Dataset rdf:about="{$resourcePrefix}/datasets/{iso19139:getResourceCode(../../.)}"> -->
++    <xsl:variable name="uuid" select="../../gmd:fileIdentifier/gco:CharacterString"/>
++    <dcat:Dataset rdf:about="{$url}/srv/fre/catalog.search#/metadata/{$uuid}">
+       <xsl:call-template name="to-dcat"/>
+     </dcat:Dataset>
+   </xsl:template>
+@@ -290,6 +304,10 @@
+     </dct:identifier>
+     <!-- xpath: gmd:identificationInfo/*/gmd:citation/*/gmd:identifier/*/gmd:code -->
+ 
++    <!-- geOrchestra addition: provide an HTML record view for the 'Voir la source originale' link -->
++    <dcat:landingPage>
++      <xsl:value-of select="$resourcePrefix"/>/<xsl:value-of select="iso19139:getResourceCode(../../.)"/>
++    </dcat:landingPage>
+ 
+     <dct:title>
+       <xsl:value-of select="gmd:citation/*/gmd:title/gco:CharacterString"/>
+@@ -299,6 +317,16 @@
+ 
+     <dct:abstract>
+       <xsl:value-of select="gmd:abstract/gco:CharacterString"/>
++
++      <!-- SIB addon-->
++      <!-- Layout will be visible only in source mode in the browser. If not in source mode, newlines won't be displayed and the keywords will be shown inline -->
```

Comment:
Actually, this section is not necessary anymore. It's adding the list of keywords in the abstract. I was doing it when data.gouv was still not supporting the keywords, but now, it's retrieving them, so I'm not ure it's still useful.

But anyway, datagouv still doesn't _show_ them in the records I believe, they're just _storing_ them and _using_ them for filtering the records.